config-service-impl/src/main/java/org/hypertrace/config/service/store/DocumentConfigStore.java

```diff
@@ -66,6 +66,12 @@ public void init(Config config) {
     this.collection = this.datastore.getCollection(CONFIGURATIONS_COLLECTION);
   }
 
+  @VisibleForTesting
```

Comment:
Pretty much always an anti-pattern. there's no DI in this module right? Suggest removing this "init" method and just have the constructor take in a Collection and a Clock. The construction of a datastore shouldn't be the responsibility of this class.